### DISCUSSION TITLE
paddle表格模型更新了？rapid ocr onnx模型有更新了吗？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/RapidTable#8] [Ideas]
🔥2024.10.1 添加OCR领域一站式全流程开发能力:

飞桨一站式全流程开发工具PaddleX，依托于PaddleOCR的先进技术，支持了OCR领域的一站式全流程开发能力：

🎨 [模型丰富一键调用](https://paddlepaddle.github.io/PaddleOCR/paddlex/quick_start.html)：将文本图像智能分析、通用OCR、通用表格识别、公式识别、印章识别涉及的17个模型整合为5条模型产线，通过极简的Python API一键调用，快速体验模型效果。此外，同一套API，也支持图像分类、目标检测、图像分割、时序预测等共计200+模型，形成20+单功能模块，方便开发者进行模型组合使用。
🚀[提高效率降低门槛](https://paddlepaddle.github.io/PaddleOCR/paddlex/overview.html)：提供基于统一命令和图形界面两种方式，实现模型简洁高效的使用、组合与定制。支持高性能部署、服务化部署和端侧部署等多种部署方式。此外，对于各种主流硬件如英伟达GPU、昆仑芯、昇腾、寒武纪和海光等，进行模型开发时，都可以无缝切换。
支持文档场景信息抽取v3([PP-ChatOCRv3](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_usage/tutorials/information_extration_pipelines/document_scene_information_extraction.md))、基于RT-DETR和PicoDet的[高精度版面区域检测模型](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/layout_detection.md)、高精度表格结构识别模型[SLANet_Plus](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/table_structure_recognition.md)、版面矫正预测模型[UVDoc](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/text_image_unwarping.md)、公式识别模型[LatexOCR](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/formula_recognition.md)、基于PP-LCNet的[文档图像方向分类模型](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/doc_img_orientation_classification.md)

**Comment by SWHL** [comment]
刚看到，还没来得及更新哈，欢迎贡献

**Comment by nissansz** [comment]
看不懂paddle模型更新在哪了，没法测试，环境跑不起来，只能等你的onnx版模型









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: SWHL  ***@***.***>
 

 
 发送时间: 2024-10-03 21:55:26
 

 
 收件人:RapidAI/RapidTable ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/RapidTable] paddle表格模型更新了？rapid ocr onnx模型有更新了吗？ (Discussion #8)
 





 刚看到，还没来得及更新哈，欢迎贡献

**Comment by cty-ai** [comment]
模型地址：https://github.com/PaddlePaddle/PaddleX/blob/878b0cc5690433ebfb74d28bc0ce9b88c5578664/paddlex/inference/utils/official_models.py#L215

**Comment by nissansz** [comment]
怎么转成rapid格式用？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Yong  ***@***.***>
 

 
 发送时间: 2024-10-10 15:06:23
 

 
 收件人:RapidAI/RapidTable ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/RapidTable] paddle表格模型更新了？rapid ocr onnx模型有更新了吗？ (Discussion #8)
 





 模型地址：https://github.com/PaddlePaddle/PaddleX/blob/878b0cc5690433ebfb74d28bc0ce9b88c5578664/paddlex/inference/utils/official_models.py#L215

**Reply by Joker1212** [reply]
已经更新了哈，因为paddle2onnx工具暂时没有支持相关算子，只能引入paddlepaddle cpu使用

**Comment by nissansz** [comment]
也没法用paddlepaddle gpu? 准确率怎么样？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: xinyi  ***@***.***>
 

 
 发送时间: 2024-10-14 08:57:33
 

 
 收件人:RapidAI/RapidTable ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/RapidTable] paddle表格模型更新了？rapid ocr onnx模型有更新了吗？ (Discussion #8)
 





 已经更新了哈，因为paddle2onnx工具暂时没有支持相关算子，只能引入paddlepaddle cpu使用

**Reply by Joker1212** [reply]
本仓库主要都是会转onnx使用的，是因为暂时转换工具有问题，临时的解决方案，使用cpu速度是很快的，准确率的问题，直接参考paddlex的文档哈

**Comment by nissansz** [comment]
paddle cpu 要用哪个版本？

我装的是paddlepaddle-gpu                              2.3.2
这个能用起来吗？








--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: xinyi  ***@***.***>
 

 
 发送时间: 2024-10-14 09:15:58
 

 
 收件人:RapidAI/RapidTable ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/RapidTable] paddle表格模型更新了？rapid ocr onnx模型有更新了吗？ (Discussion #8)
 





 本仓库主要都是会转onnx使用的，是因为暂时转换工具有问题，临时的解决方案，使用cpu速度是很快的，准确率的问题，直接参考paddlex的文档哈

**Reply by SWHL** [reply]
与其在这里问我们，自己试试不就知道了。
如果需要详细指导，请加入我们的[知识星球私享群](https://t.zsxq.com/0duLBZczw)，我们在那里会提供详尽指导。
不想加入的话，请先看看[提问的智慧](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

**Comment by nissansz** [comment]
![image](https://github.com/user-attachments/assets/591b21a3-af53-4151-84b9-7eade3c268e5)
安装后，可以用，结果好像跟之前没多大区别，表格结构有错误，没有wired2效果好
